packages/@aws-cdk/aws-appmesh/test/virtual-gateway.test.ts

```diff
@@ -116,7 +116,7 @@ describe('virtual gateway', () => {
           healthCheck: appmesh.HealthCheck.grpc(),
         })],
         mesh: mesh,
-        accessLog: appmesh.AccessLog.fromFilePath('/dev/stdout'),
+        accessLog: appmesh.AccessLog.fromFilePath('/dev/stdout', appmesh.LoggingFormat.fromText('test_pattern')),
```

Comment:
Please ensure that at least one of these unit tests still uses an unspecified or `undefined` `LoggingFormat`

Reply:
added the test